swiss-army-knife/README.md

```diff
@@ -0,0 +1,38 @@
+# Swiss-Army-Knife
+Rancher Support uses the image of a standard tool called `swiss-army-knife` to help you manage your Rancher/Kubernetes environment. You can learn more about this image by visiting its official repo at [rancherlabs/swiss-army-knife](https://github.com/rancherlabs/swiss-army-knife/)
+
+TLDR; This image has a lot of useful tools that can be used for scripting and troubleshooting.
+- [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/)
+- [`helm`](https://helm.sh/docs/intro/)
+- [`curl`](https://curl.haxx.se/docs/manpage.html)
+- [`jq`](https://stedolan.github.io/jq/)
+- [`traceroute`](https://www.traceroute.org/about.html)
+- [`dig`](https://www.dig.com/products/dns/dig/)
+- [`nslookup`](https://www.google.com/search?q=nslookup)
+- [`ping`](https://www.google.com/search?q=ping)
+- [`netstat`](https://www.google.com/search?q=netstat)
+- And many more!
+
+## Example deployments
+
+### Overlay Test
+As part of Rancher's overlay test, which can be found [here](https://rancher.com/docs/rancher/v2.6/en/troubleshooting/networking/). You can be deployed to the Rancher environment by running the following command:
+```
+kubectl apply -f https://raw.githubusercontent.com/rancherlabs/support-tools/master/swiss-army-knife/deploy/overlaytest.yaml
+```
+
+This will deploy a deamonset that will run on all nodes in the cluster. These pods will be running `tail -f /dev/null,` which will do nothing but keep the pod running.
+
```

Comment:
We could add a one-liner to run the overlay bash script as well?

eg, `curl -sfL https://raw.githubusercontent.com/rancherlabs/support-tools/master/swiss-army-knife/overlaytest.sh | bash`

Reply:
@mattmattox what do you think about this?

Reply:
added